app/views/layouts/_add_js_enabled_class_to_body.html.erb

```diff
@@ -1,3 +1,3 @@
 <script nonce='<%= content_security_policy_nonce %>'>
-  document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+  document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');
```

Comment:
I reverted this because the `add()` method omits the classes that are already present.

https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/add#return_value

<img width="1106" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/786aafe1-d0e8-4427-9918-68f746d12bc1">